javascript/javascript1/week4/voiceAssistantTask.js

```diff
@@ -0,0 +1,119 @@
+// Voice Assistant Application:
+
+let personName = "";
+const toDos = [];
+
+//Some of the motivational quotes:
+const quotes = [
+  "Keep your face always toward the sunshine, and shadows will fall behind you.",
+  "Everything you've ever wanted is sitting on the other side of fear.",
+  "Attitude is a little thing that makes a big difference.",
+  "I am not a product of my circumstances.",
+  "If you can dream it, you can do it.",
+];
+
+// For nice to meet you, benjamin.
+function getReply(command) {
+  command = command.toLowerCase();
+  if (command.startsWith("hello my name is ")) {
+    const name = command.replace("hello my name is ", "");
+    if (personName === name) {
+      return `You have already introduced yourself as ${personName}.`;
+    } else {
+      personName = name;
+      return `Nice to meet you, ${personName}.`;
+    }
+  }
+
+  // For your name is benjamin.
+  if (command === "what is my name") {
```

Comment:
This test would fail if I typed `What is my name?`.  Identifying a command using `.startWith` or `includes` is more robust.

Reply:
includes method implemented. Thank You!